app/lib/lnd/util.js

```diff
@@ -163,15 +164,17 @@ export const waitForFile = (filepath, timeout = 1000) => {
 
   // Promise B resolves when the file has been found.
   let promiseB = new Promise(resolve => {
-    let intervalId = setInterval(() => {
+    let intervalId = setInterval(async () => {
       mainLog.debug('waiting for file: %s', filepath)
-      if (!fs.existsSync(filepath)) {
+      try {
+        await stat(filepath)
+        mainLog.debug('found file: %s', filepath)
+        clearInterval(intervalId)
+        clearTimeout(timeoutId)
+        resolve()
+      } catch {
```

Comment:
is this error swallowing here intentional?

Reply:
Yes - have added additional comment